addons/web/static/src/js/views/kanban/kanban_record.js

```diff
@@ -695,9 +695,11 @@ var KanbanRecord = Widget.extend({
     _onKeyDownCard: function (event) {
         switch (event.keyCode) {
             case $.ui.keyCode.ENTER:
-                event.preventDefault();
-                this._onGlobalClick(event);
-                break;
+                if (!$(event.target).is('textarea')) {
```

Comment:
I'm not a big fan of this ; I would maybe do the opposite and only execute this if `event.target` is the kanban record itself?